theme.json

```diff
@@ -656,7 +656,8 @@
 				},
 				"color": {
 					"text": "var(--wp--preset--color--contrast)"
-				}
+				},
+				"css": " &:not(.is-style-wide):not(.is-style-dots):not(.alignwide):not(.alignfull){width: 100px}"
```

Comment:
I'd rather not include css like this when we can actually target the separator block variation [like this](https://github.com/WordPress/twentytwentyfour/blob/cd4565a27f0a48b1a94bd9307fe4ea2e97a61bdf/theme.json#L458)

Reply:
So, while we can target the `default` variation, `css` does not look like it renders for each variation. 

This for example, does not work: 

<img width="821" alt="CleanShot 2023-09-07 at 09 16 50" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/e5afe4d7-0ca9-44dd-84c3-623f2f85f2d6">

While this does: 

<img width="298" alt="CleanShot 2023-09-07 at 09 17 51" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/5466b784-db2b-425e-926f-7919769c03f4">

Reply:
Correct, CSS is not supported for elements or variations.

Reply:
> CSS is not supported for elements or variations.

I think it could be cool for variations.

Reply:
All our wishes will not be fulfilled in time for 6.4 😄 
For example, it needs a UI, otherwise, users can not manage the custom CSS.

Reply:
I wouldn't think that's a blocker—but alas, I'm not thinking custom variation css will land soon.